tests/cephfs/lib/cephfs_common_lib.py

```diff
@@ -634,3 +634,23 @@ def wait_for_two_active_mds(
                 time.sleep(retry_interval)  # Retry after the specified interval
 
         return 1
+
+    def is_ibm_build(self, test_data):
+        """
+        This method helps know if the cluster has ibmceph build installed
+        Return True if ibm-build, else False
+        """
+        if "custom-config" in test_data and isinstance(
+            test_data["custom-config"], list
+        ):
+            ibm_build_value = None
+            for config in test_data["custom-config"]:
+                key, value = config.split("=")
```

Comment:
can we make use of this if possible 
https://github.com/red-hat-storage/cephci/blob/d3d9b60aa0f291fa319c56d87ebd70fa292873ca/tests/cephfs/cephfs_utilsV1.py#L1443